src/Selenium2Library/locators/elementfinder.py

```diff
@@ -121,9 +124,108 @@ def _find_by_sc_locator(self, browser, criteria, tag, constraints):
+        for prefix in prefixes:
+            attr = prefix + 'repeat-start'
+            repeatElems = browser.execute_script("""return document.querySelectorAll('[%s]');""" % attr)
+            attr = attr.replace('\\','')
+            for elem in repeatElems:
+                if repeater_row_col['repeater'] in elem.get_attribute(attr):
+                    item = elem
+                    is_end = item.get_attribute('ng-repeat-end')
+                    row = []
+                    while is_end!='':
+                        row.append(item)
+                        try:
+                            is_end = item.get_attribute('ng-repeat-end')
+                            item = item.find_element_by_xpath('./following-sibling::*')
+                        except:
+                            item = None
+                            is_end = ''
+                    multiRows.append(row)
+
+        # if ...@row[index]...
+        if repeater_row_col['row_index'] is not None:
+            if rows:
+                rows = get_iterable(rows[repeater_row_col['row_index']])
+            if multiRows:
+                multiRows = get_iterable(multiRows[repeater_row_col['row_index']])
+
+        # if ...@col=binding... 
+        if repeater_row_col['col_binding'] is not None:
+            from selenium.common.exceptions import NoSuchElementException
+            bindings=[]
+            for row in rows:
+                if 'ng-binding' in row.get_attribute("class"):
+                    bindings.append(row)
+                try:
+                    childBindings = row.find_elements_by_class_name('ng-binding')
+                    bindings.extend(childBindings)
+                except NoSuchElementException:
+                    pass
+            
+            for row in multiRows:
+                row = get_iterable(row)
+                for elem in row:
+                    if 'ng-binding' in elem.get_attribute("class"):
+                        bindings.append(elem)
+                    try:
+                        childBindings = elem.find_elements_by_class_name('ng-binding')
+                        bindings.extend(childBindings)
+                    except NoSuchElementException:
+                        pass
+            
+            for bind in bindings:
+                bindingName = browser.execute_script("dataBinding=angular.element(arguments[0]).data('$binding');if(dataBinding){bindingName=dataBinding.exp||dataBinding[0].exp||dataBinding;}return bindingName;",bind)
+                if repeater_row_col['col_binding'] in bindingName:
+                    matches.append(bind)
+            return matches
+        
+        matches = rows
+        matches.extend(multiRows)
+        return matches
+
     def _find_by_default(self, browser, criteria, tag, constraints):
         if criteria.startswith('//'):
             return self._find_by_xpath(browser, criteria, tag, constraints)
+        elif criteria.startswith('{{'):
```

Comment:
Ah, now this is pretty neat. I should've definitely added this as a part of the custom locator thing.... I'll go make an issue for that.